actnow/petitions/models.py

```diff
@@ -63,6 +63,7 @@ class Signature(TimestampedModelMixin):
         max_length=255,
         blank=True,
     )
+    anonymous = models.BooleanField(verbose_name=_("Anonymous"), default=False)
```

Comment:
```suggestion
    anonymous = models.BooleanField(verbose_name=_("anonymous"), default=False)
```
Or why are we capitalizing it?